server/server.go

```diff
@@ -642,3 +660,45 @@ func checkBranchTranslationCmdF(args []string, w http.ResponseWriter, slashComma
 
 	return nil
 }
+
+func pipelineTriggerCmdF(args []string, w http.ResponseWriter, slashCommand *MMSlashCommand) error {
+	const colorErr = "#ee2116"
+	const colorSuccess = "#0060aa"
+
+	if len(args) < 1 {
```

Comment:
nit: more idiomatic to write:
```suggestion
	if len(args) == 0 {
```